apps/greencheck/tests/test_importer_csv.py

```diff
@@ -0,0 +1,83 @@
+import pytest
+import pathlib
+import re
+import pandas as pd
```

Comment:
can you please make sure pandas is added to the pipfile?

```
pipenv install pandas
```

you may need to familiarise yourself with pipenv if you haven't read up on the basis usage already

https://pipenv.pypa.io/en/latest/basics/